sdcm/utils/remote_logger.py

```diff
@@ -0,0 +1,135 @@
+
+    def __init__(self, node, target_log_file: str):
+        super().__init__(node, target_log_file)
+        self._termination_event = Event()
+        self._remoter = node.remoter
+        self._thread = Process(target=self._journal_thread, daemon=True)
+        if self._logger_cmd is None:
+            raise ValueError("_logger_cmd should be overridden")
+
+    def _file_exists(self, file_path):
+        try:
+            result = self._remoter.run('sudo test -e %s' % file_path,
+                                       ignore_status=True)
+            return result.exit_status == 0
+        except Exception as details:  # pylint: disable=broad-except
+            self._log.error('Error checking if file %s exists: %s',
+                            file_path, details)
+
+    def _log_retrieve(self, since):
+        if not since:
+            since = 'the beginning'
+        self._log.debug(self._retrieve_message.format(since=since))
+
+    def _retrieve(self, since):
+        since = '--since "{}" '.format(since) if since else ""
+        self._remoter.run(self._logger_cmd.format(since=since),
+                          verbose=True, ignore_status=True,
+                          log_file=self._target_log_file)
+
+    def _retrieve_journal(self, since):
+        try:
+            self._log_retrieve(since)
+            self._retrieve(since)
+        except Exception as details:  # pylint: disable=broad-except
+            self._log.error('Error retrieving remote node DB service log: %s', details)
+
+    @raise_event_on_failure
+    def _journal_thread(self):
+        read_from_timestamp = None
+        while not self._termination_event.is_set():
+            self._wait_ssh_up(verbose=False)
+            self._retrieve_journal(since=read_from_timestamp)
+            read_from_timestamp = datetime.utcnow().strftime("%Y-%m-%d %H:%M:%S")
+
+    def _wait_ssh_up(self, verbose=True, timeout=500):
+        text = None
+        if verbose:
+            text = '%s: Waiting for SSH to be up' % self
+        wait.wait_for(func=self._remoter.is_up, step=10, text=text, timeout=timeout, throw_exc=True)
+
+    def start(self):
+        self._thread.start()
+
+    def stop(self, timeout=None):
+        self._thread.terminate()
+        self._thread.join(timeout)
+        if self._thread.is_alive():
+            self._thread.kill()  # pylint: disable=no-member
+
+
+class SSHScyllaSystemdLogJournal(SSHLogJournalBase):
+    _logger_cmd = 'sudo journalctl -f --no-tail --no-pager --utc {since} ' \
+                  '-u scylla-ami-setup.service '\
+                  '-u scylla-image-setup.service '\
+                  '-u scylla-io-setup.service '\
+                  '-u scylla-server.service '\
+                  '-u scylla-jmx.service'
+
+
+class SSHScyllaFileLogJournal(SSHLogJournalBase):
+    _logger_cmd = 'sudo tail -f /var/log/syslog'
+    # _logger_cmd = 'sudo tail -f /var/log/syslog | grep scylla'
+
+
+class SSHCassandraFileLogJournal(SSHLogJournalBase):
```

Comment:
I don't think we can actually run cassandra, so no much point adding this code

Reply:
yes please remove, we don't support Cassandra